react/src/components/FieldDisplayEditable.tsx

```diff
@@ -131,8 +131,11 @@ function EnhancedTextField({
             // Value is typed in, and can be really long (notes)
             textFieldProps.multiline = true;
             textFieldProps.maxRows = 2;
+        } else if (field.fieldName === "month_of_birth") {
+            textFieldProps.InputLabelProps = { shrink: true };
+            textFieldProps.type = "month";
```

Comment:
So there's no way to get the validation code to work with `type="month"` in Firefox?

Reply:
Type "month" is not supported in Firefox. (Type "date" is supported in Firefox, but not in Internet Explorer.) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month

Reply:
When you set an unsupported type for an input element, it simply falls back to a text field. So by default, this control shouldn't become unusable in Firefox (where I wasn't able to type in anything if the field started out blank). This is why I think it's something to do with the validation code. We can certainly work with just a text field in all browsers, but the point of the fallback behaviour of input elements is so that you can have fancy stuff if the browser supports it (in this case the built-in month selector) and gracefully fall back if there's no browser support, instead of completely breaking.